content/docs/dvclive/get-started.md

```diff
@@ -16,25 +17,37 @@ live = Live()
 
 See [`Live()`](/doc/dvclive/api-reference/live) for details.
 
-### 2. Log metrics
+### Log data
 
 ```python
-live.log(metric_name, value)
+live.log("acc", 0.9)
 ```
 
 See `Live.log()` for details.
 
-### 3. Increase the step number
+### (Optionally) Update the step number
 
 ```python
 live.next_step()
 ```
 
-See `Live.next_step()` for details.
+See `Live.next_step()` and `Live.set_step()` for details.
+
+### No step
+
+In ML scenarios where there are no iterations (steps) as part of the training
+process, there is no need to update the step number.
+
+When you don't update the step number in your code, DVCLive will **not**
+generate:
+
+- [linear plots](/doc/dvclive/get-started#linear-plots)
```

Comment:
> - What was wrong with `metrics logs`?

I just find `metrics logs` or the other alternatives are a little abstract. I think that "data series", "scalar plots", or line/linear plots better indicate what it's being saved.

Reply:
Without `dvc plots`, they are `.tsv` files, and that's what's actually being saved. IMO plots is a bit unnatural and DVC-specific. It also seems to me like the key feature of these are that they capture the metrics over time at each step, showing training progression. What would describe that? What about something like "step `.tsv` files"? I don't really like it but not sure what's better.

Do you feel the same about "metrics summary"? That seems similarly abstract to me.

Reply:
> It also seems to me like the key feature of these are that they capture the metrics over time at each step, showing training progression. What would describe that?

Metrics history? 

> Do you feel the same about "metrics summary"? That seems similarly abstract to me.

I don't know why but I don't feel the same for summary. Maybe because I can't think of alternatives 😅

Reply:
I think "metrics history" is pretty good. What do you think?

Reply:
Updated with metrics history. Overall I like how it reads

Reply:
I don't see how "metrics logs" is abstract. It's pretty concrete and specific to what those TSV files are. It's just a bit confusing in the current Get Started since there's a "Log Metrics" section and later a "Metrics Logs" subsection. Probably there's no need for those subsections (under **Outputs**): just use paragraphs, then the terms don't become titles. "Metrics logs" does also read funny because of the final s in both words though so maybe be more descriptive like "TSV log files of your metrics" or something like that.

"Metrics history" to me sounds like it could relate to the Git history i.e. how metrics have evolved between commits.

Reply:
For me, `history` makes more sense than `logs` when combined with the other output, named `summary`. It feels like `summary` and `history` are "connected".

`history` is also a term [used in Keras ](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/History) with similar meaning as here, so data scientist might be already familiar with it.

> Probably there's no need for those subsections (under **Outputs**): just use paragraphs, then the terms don't become titles.

In subsequent P.R. I will remove the outputs subsections https://dvc-org-dvclive-data-hpgunbngf.herokuapp.com/doc/dvclive/get-started#outputs